tests/process_monitor/test_process_monitor.py

```diff
@@ -296,7 +296,7 @@ def test_MantarrayProcessesMonitor__handled_completed_mag_analysis_command_corre
     invoke_process_run_and_check_errors(monitor_thread)
     assert is_queue_eventually_empty(data_analyzer_to_main) is True
 
-    ws_message = queue_to_server_ws.get(timeout=QUEUE_CHECK_TIMEOUT_SECONDS)
+    ws_message = queue_to_websocket_ws.get(timeout=QUEUE_CHECK_TIMEOUT_SECONDS)
```

Comment:
Be careful when doing mass find + replace, this name should just be `queue_to_websocket`. I think I saw this happen in some other files as well